CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@ Thanks for considering making a contribution to the IOTA network or its document
 
 ## Contribute to IOTA
 
-See [IOTA Environment Setup](https://github.com/iotaledger/iota/blob/main/docs/content/guides/developer/getting-started/iota-environment.mdx) for approach to submitting code fixes and enhancements.
+See [IOTA Environment Setup](https://github.com/iotaledger/iota/blob/main/docs/content/developer/getting-started/iota-environment.mdx) for approach to submitting code fixes and enhancements.
```

Comment:
```suggestion
See [IOTA Environment Setup](https://github.com/iotaledger/iota/blob/develop/docs/content/developer/getting-started/iota-environment.mdx) for approach to submitting code fixes and enhancements.
```

Reply:
Oh I just saw your description! I think it's fine to change 👍🏻

Reply:
Just to be sure, we keep it `main` or not? (I am guessing, `develop` will not be re-directed, if we change our default branch in the future.. )

Reply:
I think we should change to `develop` yes, we've already changed a lot of main to develop, looks like this one slipped

Reply:
I've seen there are still lots left so I've created https://github.com/iotaledger/iota/issues/4604.
Feel free to resolve this discussion without addressing it and it could be done as part of this issue.